client/src/components/composite/LoginForm/LoginForm.tsx

```diff
@@ -57,6 +57,7 @@ const LoginForm = ({
     if (isLoading || !loginHandler) return
     try {
       setIsLoading(true)
+      alert("Logging In...")
```

Comment:
we want something to display under the button, not an alert (which blocks everyting)